app/routes.py

```diff
@@ -1,2 +1,69 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.planet import Planet
 
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods = ["POST"])
+def handle_planets():
+    request_body = request.get_json()
+    new_planet = Planet(
+        name = request_body["name"], 
+        description = request_body["description"], 
+        distance_from_sun = request_body["distance from sun"]
+        )
+
+    db.session.add(new_planet)
+    db.session.commit()
+
+    return make_response(jsonify(f'planet {new_planet.name} successfully created!'), 201)
+
+@planets_bp.route("", methods = ["GET"])
+def planet_data():
+    planets = Planet.query.all()
+    planets_response = []
+
+    for planet in planets:
+        planets_response.append(planet.to_dictionary())
```

Comment:
Nothing wrong with this, but it's good practice for list comprehensions to use them when we're looking to fill a list:
```suggestion
    planets_response = [planet.to_dictionary() for planet in planets]
```